rita_exit/src/traffic_watcher/mod.rs

```diff
@@ -244,6 +281,35 @@ pub fn watch(
         }
     }
 
+    for (wg_key, bytes) in new_counters.clone() {
+        let state = (
```

Comment:
there's no need to duplicate all this logic, just add a little routine to merge new counters with counters summing the amounts and then you can avoid duplicating these blocks.

Reply:
fixed